task2/InterfaceManager.py

```diff
@@ -0,0 +1,109 @@
+import json
+import logging
+from dataclasses import dataclass, field
+from typing import List
+
+import marshmallow as mm
+import requests
+from dataclasses_json import DataClassJsonMixin, config
+from marshmallow import fields, EXCLUDE
```

Comment:
@patrikprovaznik do you have these in requirements file?

Reply:
@patrikprovaznik you are actually missing readme and requirements file.. please add that in